README.md

```diff
@@ -1 +1,18 @@
 # dandisets-linkml-status
+
+[![PyPI - Version](https://img.shields.io/pypi/v/dandisets-linkml-status.svg)](https://pypi.org/project/dandisets-linkml-status)
+[![PyPI - Python Version](https://img.shields.io/pypi/pyversions/dandisets-linkml-status.svg)](https://pypi.org/project/dandisets-linkml-status)
+[![Hatch project](https://img.shields.io/badge/%F0%9F%A5%9A-Hatch-4051b5.svg)](https://github.com/pypa/hatch)
+
+-----
+
+## Table of Contents
+
+- [License](#license)
+
+## License
+
+
+[//]: # (todo: make sure this is the correct license. Update LICENSE.TXT accordingly as well)
```

Comment:
@yarikoptic Can you help input the correct license info. The current info is generated by Hatch mostly. I am perfectly fine the direct the license to DANDI. I just don't know the proper way to put it.